apis/slo/v1alpha1/nodeslo_types.go

```diff
@@ -177,8 +177,10 @@ type ResourceThresholdStrategy struct {
 
 	// if be CPU RealLimit/allocatedLimit > CPUEvictBESatisfactionUpperPercent, then stop evict BE pods
 	CPUEvictBESatisfactionUpperPercent *int64 `json:"cpuEvictBESatisfactionUpperPercent,omitempty"`
-	// if be CPU (RealLimit/allocatedLimit < CPUEvictBESatisfactionLowerPercent and usage nearly 100%) continue CPUEvictTimeWindowSeconds,then start evict
+	// if be CPU (RealLimit/allocatedLimit < CPUEvictBESatisfactionLowerPercent and usage >= CPUEvictBEUsageHighEnough) continue CPUEvictTimeWindowSeconds,then start evict
 	CPUEvictBESatisfactionLowerPercent *int64 `json:"cpuEvictBESatisfactionLowerPercent,omitempty"`
+	// if be CPU (RealLimit/allocatedLimit < CPUEvictBESatisfactionLowerPercent and usage >= CPUEvictBEUsageHighEnough) continue CPUEvictTimeWindowSeconds,then start evict
+	CPUEvictBEUsageHighEnough *int64 `json:"cpuEvictBEUsageHighEnough,omitempty"`
```

Comment:
to make the naming more concise, how about `CPUEvictBEUsageHighPercent`

Reply:
i think HighEnough makes sence, it's ok for me to add suffix Percent making it `CPUEvictBEUsageHighEnoughPercent`

Reply:
Alright. WDYT @jasonliu747

Reply:
FMPOV, the word `Threshold` would fit the scenario.

Reply:
`CPUEvictBEUsageThresholdPercent` or `CPUEvictBEUsageThreshold` ?

Reply:
sorry for the late response. `ThresholdPercent` SGTM.